erpnext/hr/doctype/leave_application/leave_application.py

```diff
@@ -41,6 +42,67 @@ def validate(self):
 	def on_submit(self):
 		self.validate_back_dated_application()
 
+	def validate_parental_leave(self):
+		if self.leave_type:
+			leave_type = frappe.get_doc("Leave Type", self.leave_type)
+			if leave_type.is_parental_leave == 1:
+				# Check on Applicable After
+				if leave_type.applicable_after > 0:
```

Comment:
validate applicable_after and max_leaves_allowed for all leave_types, don't restrict to parental leave